gnome-shell/src/light/meson.build

```diff
@@ -0,0 +1,83 @@
+# destination directory
+install_dir = join_paths(gnomeshell_theme_dir, meson.project_name())
```

Comment:
As I commented in https://github.com/ubuntu/yaru/issues/1483#issuecomment-544180979, I believe it's better here to keep symmetry with what you're doing for the dark theme, that is:

```
install_dir = join_paths(gnomeshell_alt_theme_dir, meson.project_name() + '-light', 'gnome-shell')
``` 

and then to invert the symlinking in `install-shell.sh`. This better generalizes to future variants, also. And it follows the time-tested pattern of symlinking the chosen variant between a set of configuration options.